### ISSUE TITLE
Shortcuts don't work captialized 

### ISSUE THREAD

**Issue by alv1n** [dralletje/Windowed#31]
When caps lock is enabled (or shift is pressed), the shortcuts f,i,w do not work anymore. 

I'm using the plugin on firefox, though I think that shouldn't matter.

**Comment by bearoxo** [comment]
yeah, it doesn't work on chrome too. it gets funky when CAPS is on.

the shortcuts also doesn't work when CAPS is off. 
F opens up the dialog box thingy instead of going straight full screen. only then you can use the shortcut keys properly.
I goes into PiP instead of In-window.
W pops up "Caption looks like this" in the video. I'm guessing it probably because the shortcut key is conflicting with something.
P doesn't do anything.

**Comment by dralletje** [comment]
> F opens up the dialog box thingy instead of going straight full screen. only then you can use the shortcut keys properly.
You can only use the shortcuts when the popup is open. When the popup is not open, this extension respects the shortcuts the website has. This is not a mistake, this is completely by design 😁

**Comment by dralletje** [comment]
I'm leaving the shortcuts on ignored when using shift/caps lock.
